content/en/docs/guides/interceptors.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Interceptors
+description: >-
+  Explains how interceptors can be used for implementing generic behavior that
+  applies to many RPC methods.
+---
+
+### Overview
+
+The core of making gRPC services is implementing RPC methods. But some
+functionality is independent of the method being run and should apply to all or
+most RPCs. Interceptors are well suited to this task.
+
+### When to Use Interceptors
+
+You may already be familiar with the concept of interceptors, but may be used to
+calling them "filters" or "middleware." Interceptors are very well suited to
+implementing logic that is not specific to a single RPC method. They are also
+easy to share across different clients or servers. Interceptors are an important
+and frequently-used way to extend gRPC. You might find some functionality you
+want is already available as an interceptor in the wider gRPC ecosystem.
+
+Some example use cases for interceptors are:
+
+ * [Metadata](/docs/guides/metadata/) handling
+ * Logging
+ * Fault injection
+ * Caching
+ * Metrics
+ * Policy enforcement
+ * Server-side Authentication
+ * Server-side Authorization
+
+While _client-side_ authentication could be done via an interceptor, gRPC
+provides a specialized "call credentials" API that is better suited to the task.
+See the [Authentication Guide](/docs/guides/auth/) for details about client-side
+authentication.
```

Comment:
This section feels like a "callout", want to wrap this section with this?

{{% alert title="Note" color="info" %}}
...
{{% /alert %}}